benchmarks/src/gatling/scala/org/apache/polaris/benchmarks/simulations/CreateCommits.scala

```diff
@@ -0,0 +1,138 @@
+import org.apache.polaris.benchmarks.parameters.{
+  ConnectionParameters,
+  DatasetParameters,
+  WorkloadParameters
+}
+import org.apache.polaris.benchmarks.util.CircularIterator
+import org.slf4j.LoggerFactory
+
+import java.util.concurrent.atomic.{AtomicBoolean, AtomicReference}
+import scala.concurrent.duration._
+
+class CreateCommits extends Simulation {
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  // --------------------------------------------------------------------------------
+  // Load parameters
+  // --------------------------------------------------------------------------------
+  val cp: ConnectionParameters = config.connectionParameters
+  val dp: DatasetParameters = config.datasetParameters
+  val wp: WorkloadParameters = config.workloadParameters
+
+  // --------------------------------------------------------------------------------
+  // Helper values
+  // --------------------------------------------------------------------------------
+  private val accessToken: AtomicReference[String] = new AtomicReference()
+  private val shouldRefreshToken: AtomicBoolean = new AtomicBoolean(true)
+
+  private val authActions = AuthenticationActions(cp, accessToken)
+  private val tableActions = TableActions(dp, wp, accessToken)
+  private val viewActions = ViewActions(dp, wp, accessToken)
+
+  // --------------------------------------------------------------------------------
+  // Authentication related workloads:
+  // * Authenticate and store the access token for later use every minute
+  // * Wait for an OAuth token to be available
+  // * Stop the token refresh loop
+  // --------------------------------------------------------------------------------
+  val continuouslyRefreshOauthToken: ScenarioBuilder =
+    scenario("Authenticate every minute using the Iceberg REST API")
+      .asLongAs(_ => shouldRefreshToken.get())(
+        feed(authActions.feeder())
+          .exec(authActions.authenticateAndSaveAccessToken)
+          .pause(1.minute)
+      )
+
+  val waitForAuthentication: ScenarioBuilder =
+    scenario("Wait for the authentication token to be available")
+      .asLongAs(_ => accessToken.get() == null)(
+        pause(1.second)
+      )
+
+  val stopRefreshingToken: ScenarioBuilder =
+    scenario("Stop refreshing the authentication token")
+      .exec { session =>
+        shouldRefreshToken.set(false)
+        session
+      }
+
+  // --------------------------------------------------------------------------------
+  // Read and write workloads:
+  // * Create table commits by updating table properties
+  // * Read namespaces, tables and views
+  // --------------------------------------------------------------------------------
+  val tableUpdateScenario: ScenarioBuilder =
+    scenario("Create table commits by updating properties")
+      .exec(authActions.restoreAccessTokenInSession)
+      .feed(tableActions.propertyUpdateFeeder())
+      .exec(tableActions.updateTable)
```

Comment:
I guess this makes a single table commit. It would be nice to have a multi-table commit simulation too (eventually) :)

Reply:
Updating properties in multiple tables at the same time (the `/v1/{prefix}/transactions/commit` endpoint) is realistic enough for now :) That should exercise the code path that deals with atomically changing multiple entities in Persistence (same Persistence operation as for table data updates).